src/lib/InstantSearch.ts

```diff
@@ -256,21 +281,25 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
     };
```

Comment:
line above this requires https://github.com/algolia/algoliasearch-helper-js/pull/739